plugin/pkg/scheduler/factory/factory.go

```diff
@@ -338,6 +338,11 @@ func (f *ConfigFactory) CreateFromConfig(policy schedulerapi.Policy) (*scheduler
 			}
 		}
 	}
+	// Providing HardPodAffinitySymmetricWeight in the policy config is the new and preferred way of providing the value.
+	// Give it higher precedence when it is provided.
```

Comment:
Yes, that's also what I mean https://github.com/kubernetes/kubernetes/blob/master/plugin/cmd/kube-scheduler/app/options/options.go#L44 , perhaps we can update the comment a bit by saying `higher precedence than scheduler CLI configuration`.